bin/metadata_merge

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import click
+import json
+
+@click.command()
+@click.argument(
+    'metadata',
+    type=click.Path(
+        dir_okay=False,
+        file_okay=True,
+        writable=False,
+        exists=True,
+    ),
+    required=True
+)
+@click.argument(
+    'schema',
+    type=click.Path(
+        dir_okay=False,
+        file_okay=True,
+        writable=True,
+        exists=True,
+    ),
+    required=True
+)
+def main(metadata, schema):
```

Comment:
@jklukas is this how you envisioned metadata being merged?

Reply:
Yes, this looks entirely reasonable.